test/runtests.jl

```diff
@@ -299,7 +309,8 @@ macro test_enumerate(input)
 end
 
 @test_enumerate [:a, :b, :c]
-@test_enumerate []
+@test_enumerate Union{}[]
+@test_enumerate Any[]
```

Comment:
Note that on 0.4+ `Any[]` is the same with `[]`. Doesn't really mater which one to use.